lib/src/backend/declarer.dart

```diff
@@ -41,6 +41,9 @@ class Declarer {
   /// Whether to collect stack traces for [GroupEntry]s.
   final bool _collectTraces;
 
+  /// Whether to retry failed tests when configured.
```

Comment:
This is a little vague. Maybe "Whether to retry tests with the `retry` option set.".